recipes/robotraconteur/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
+
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+    mkdir -p bluez/include/bluetooth
+    cp -r bluez-src/lib/*.h bluez/include/bluetooth/
+    CXXFLAGS="$CXXFLAGS -I$SRC_DIR/bluez/include"
+fi
+
+mkdir -p build2
+cd build2
+
+cmake -DBUILD_TESTING=OFF -DBUILD_GEN=ON -DBUILD_PYTHON3=ON \
+    -DBUILD_NET=ON -DRR_NET_BUILD_NATIVE_ONLY=ON \
+    -DRR_NET_INSTALL_NATIVE_LIB=ON -DBUILD_DOCUMENTATION=OFF \
+    -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX -DCMAKE_PREFIX_PATH:PATH=$PREFIX \
```

Comment:
```suggestion
    ${CMAKE_ARGS} \
```

Reply:
Can you show exactly what cmake args you require?